### PR TITLE
Allowing magic for people not running locally

### PR DESCRIPTION
**Pull request by jrgleason**
This provides a few make commands and some docs to make deploying to non-local examples easier.

### CODE REVIEW

**Comment by jrgleason** [comment]
@LukeLalor Is eidolon an ok default namespace. I don't think it is good to use the default.

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,10 @@
-DOCKER_REPO_NAME := my-eidolon-project
+DOCKER_REPO_URL ?=
+DOCKER_REPO_NAME := $(if $(DOCKER_REPO_URL),$(DOCKER_REPO_URL),my-eidolon-project)
```
Comment (LukeLalor) [inline Makefile]: Do we need to redefining this? can we use DOCKER_REPO_NAME ?= my-eidolon-project to set the default?

File: Makefile [hunk 2]
```diff
@@ -117,37 +119,54 @@ check-install-operator:
 k8s-serve: k8s-server k8s-webui
 	@echo "Press Ctrl+C to exit"
 	@echo "------------------------------------------------------------------"
-	@echo "Server is running at $$(./k8s/get_service_url.sh eidolon-ext-service)"
-	@echo "WebUI is running at $$(./k8s/get_service_url.sh eidolon-webui-service)"
+	@echo "Server is running at $$(./k8s/get_service_url.sh eidolon-ext-service --namespace=$(NAMESPACE))"
+	@echo "WebUI is running at $$(./k8s/get_service_url.sh eidolon-webui-service --namespace=$(NAMESPACE))"
```
Comment (LukeLalor) [inline Makefile:122]: This did not work for me

**Review by LukeLalor — CHANGES_REQUESTED** [review]
Mostly works for me with with colima.

We need to update the get_service_url scripts, but everything else seems to be working 👍

File: Makefile (hunk 2 above)
Comment (LukeLalor) [reply Makefile:122]:
```
------------------------------------------------------------------
Error from server (NotFound): services "eidolon-ext-service" not found
Server is running at Error: Couldn't find NodePort for 'eidolon-ext-service'. Make sure the service exists and is of type NodePort.
Error from server (NotFound): services "eidolon-webui-service" not found
WebUI is running at Error: Couldn't find NodePort for 'eidolon-webui-service'. Make sure the service exists and is of type NodePort.
------------------------------------------------------------------
```

Comment (jrgleason) [reply Makefile:122]: Interesting could you tell why it failed? Adding the namespace is something that can be removed I just didn't like using the default namespace.

File: Makefile (hunk 1 above)
Comment (jrgleason) [reply Makefile]: It does default this is the repo url in case it isn't local

File: Makefile (hunk 2 above)
Comment (LukeLalor) [reply Makefile:122]: I don't think the bash script accepts the --namespace argument. I didn't debug it beyond the log statement above

Comment (jrgleason) [reply Makefile:122]: @LukeLalor Can we work together to get this to work with the namespaces? I think there is just some issue maybe with it already being installed. Lets walk through it and see what the issue is.

**Review by LukeLalor — APPROVED** [review]

**Pull request merged**